helm-charts/snapshot/templates/cr.yaml

```diff
@@ -0,0 +1,29 @@
+{{- range $path, $_ :=  (.Files.Glob "files/*:*.*") }}
```

Comment:
It will be also good to check the Radme file to rename crd - cr where it's needed.

Reply:
BTW, controller works with CRs and CRDs